Goal: installing smbmount and smbumount is done The Debian Way in debian/rules

Fixes: ?

Status wrt upstream: Debian specific

Note: 

Index: samba-3.0.25b/source/script/installbin.sh.in
===================================================================
--- samba-3.0.25b.orig/source/script/installbin.sh.in	2007-06-27 09:25:37.244159369 +0200
+++ samba-3.0.25b/source/script/installbin.sh.in	2007-06-27 09:28:30.235959381 +0200
@@ -21,13 +21,15 @@
  chmod $INSTALLPERMS $DESTDIR/$BINDIR/$p2
 
  # this is a special case, mount needs this in a specific location
- if [ $p2 = smbmount ]; then
-   if [ ! -d $DESTDIR/@rootsbindir@ ]; then
-      mkdir $DESTDIR/@rootsbindir@
-   fi
-   echo "Creating sym link $DESTDIR/@rootsbindir@/mount.smbfs to $BINDIR/$p2 "
-   ln -sf $BINDIR/$p2 $DESTDIR/@rootsbindir@/mount.smbfs
- fi
+ # Commented out for the Debian Samba package. We take care of this
+ # important symlink in debian/rules. (peloy@debian.org)
+ # if [ $p2 = smbmount ]; then
+ #  if [ ! -d $DESTDIR/@rootsbindir@ ]; then
+ #     mkdir $DESTDIR/@rootsbindir@
+ #  fi
+ #  echo "Creating sym link $DESTDIR/@rootsbindir@/mount.smbfs to $BINDIR/$p2 "
+ #  ln -sf $BINDIR/$p2 $DESTDIR/@rootsbindir@/mount.smbfs
+ # fi
 done
 
 
